documentation/solution-architecture.md

```diff
@@ -0,0 +1,151 @@
+
+As NLX aims at conforming to EIF/EiDAS and by having the waardepapieren solution use this platform for information retrieval and the NLX certificates for signing (and validation) and by externalizing the identification / authentication and partly even authorisation of actors, this solution quite automatically adheres to EIF/EiDAS.
+
+As already mentioned, the solution does not store any personal information on ledgers. All information retrieved through NLX are only held temporarily in memory to be able to issue the information to the holder of the requested credential, often being the subject of the information. Though in practice the holders also can be actors that are allowed to retrieve proofs for subjects other then themselves, for instance notaries, we focus on the case where the holder is the subject itself and defer other types of holders to the integration projects of the waardepapieren solution. The reason for this is that Discipl can easily support these alternative authorizations. By doing so, as long as the issuer and validating actor complies to GDPR in relation to the information subject, so does this solution. The waardepapieren solution helps complying to GDPR by holding information only temporarily and through other measures. The software is open source and is to be independently audited on the topic of security.
+
+As for the ETSI TS 103 171 standard we see a need for a similar standard based on JSON and possibly one that better aligns with a data landscape that holds public keys in official distributed ledgers in the open apart from signed data shared privately. Though it is easy to create a XaDES export of any verifiable proof created from the waardepapieren solution, we do not see a direct need for this however for this project in it's current stage and also defer this topic to ongoing development of Discipl.
+
+By using NLX certificates for signing and information retrieval, the waardepapieren service will align with REGULATION (EU) No 910/2014 for a large extend, the rest of which is to be deferred to the ongoing development of Discipl like as with ETSI TS 103 171.
+
+The Waardepapieren solution should adhere to EN 301 549 / WCAG 2.0 though this is largely dependent on the integration of the solution in the user interfaces at the side of the issuer and not the solution itself. The project does involve the implementation of a reference validator app  / wallet however which must adhere to EN 301 549 / WCAG 2.0.
+
+As said, the solution also aims at a W3C verifiable credential compliant solution. This compliancy is deferred to the ongoing development of Discipl however. The solution adheres to the main mandatory concepts of this and related standards however.
+
+## Organisational layer
+
+The following stakeholders are identified:
+
+- issuer : automated (abundance) service available at location of issuer and/or through its website
```

Comment:
The abundance service library is a library for setting up services that are set up to focus on attending to needs in a standard way and such type of services are coined "abundance services" hence the name of the library.. A component is never the one and only abundance service but a specific service which happens to be of an abundance service type because it conforms to this idea of primarily attending to needs by using the abundance service library... so an abundance service is a service with a specific type of focus, the abundance service library helps you to create such type of services. I don see the problem in this. The term could be defined more explicitely somewhere in the document though.  

And maybe it is better to talk about:

- issuer : Entity that issues verifiable claims. Also refers to an automated (abundance type of) service available at location of the issuer and/or through its website to act on behalf of the issuer.

Reply:
changed it like that